src/xdocs/config_filters.xml

```diff
@@ -986,5 +986,277 @@ public class UserService {
           <p> <a href="config.html#TreeWalker">TreeWalker</a> </p>
         </subsection>
       </section>
+
+    <section name="SuppressWithPlainTextCommentFilter">
```

Comment:
in this comment, I proposed this name - https://github.com/checkstyle/checkstyle/issues/4841#issuecomment-320483860

but after site generation I read names:
SuppressWithNearbyCommentFilter
SuppressWithPlainTextCommentFilter

and I was not sure what this filter does .....
`SuppressWithNearbyCommentFilter` - looks like AST based filter as part of language `comment` is in name.
`SuppressWithPlainTextCommentFilter` - why "nearby" is missed ? why `comment` is present  but user can define what ever he wants to be a marker to suppress (not only by comment)?

Looks like name should be `SuppressWithNearbyPlainTextFilter`.
@rnveach , please share your ideas.

Reply:
@romani This is not `Nearby` because it defines start and end areas where checkstyle violations are suppressed.
`Nearby` looks for a comment +/- X lines from violation to know to suppress it.

I don't feel this should be named `Nearby` as it will confuse people with the real nearby.

Reply:
good point, as we do analog of SuppressionCommentFilter
http://checkstyle.sourceforge.net/config_filters.html#SuppressionCommentFilter

name `SuppressWithPlainTextCommentFilter` looks ok.
Looks like "with" mapped by mind to not right filter.

@MEZk , please skip this review point.